tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -710,6 +721,36 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
         )
     return is_valid, error
 
+
+def t3_m3_child_validator(which_child):
+    """T3 child validator."""
+    def t3_first_child_validator_func(value, temp, friendly_name, item_num):
+        if not _is_empty(value, 1, 60) and len(value) >= 60:
+            return (True, None)
+        else:
+            return (False, "1st child record truncated.")
+
+    def t3_second_child_validator_func(value, temp, friendly_name, item_num):
+        if not _is_empty(value, 60, 101) and len(value) >= 101 and not _is_empty(value, 8, 19):
+            return (True, None)
+        else:
+            return (False, "2nd child record truncated.")
```

Comment:
I can't remember if I asked in the OG PR, but is this the error language Alex and Miles provided?

Reply:
no, I pinged Alex and Miles in the other PR for review